Unix/http/httpcommon.c

```diff
@@ -214,6 +214,12 @@ MI_Result CreateSSLContext(SSL_CTX **sslContext, SSL_Options sslOptions)
         options |= SSL_OP_NO_TLSv1_2;
     }
 #endif
+#ifdef SSL_OP_NO_COMPRESSION
```

Comment:
SSL_OP_NO_COMPRESSION  is an option from the SSH header file itself, in much the same way that SSL_OP_NO_TLSv1_2 and the other one is. I duplicated the code that was done for other options, so it is consistent with that (I believe). We process the option from the config file, then when we are try to use it we see if SSL actually has the option available in the include file. Not all platforms so. Not all platforms have SSL_OP_NO_TLSv1_2 flag in the SSL header file either which is why this is a duplicate of that code.